app/models/review.rb

```diff
@@ -1,4 +1,4 @@
 class Review < ApplicationRecord
-  validates_presence_of :title
+  validates_presence_of :title, :rating, :content
```

Comment:
Must figure out how to test flash message appear with these blocking edit due to relationship.